### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by kalimullina**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw1/TestBase.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.testng.annotations.BeforeMethod;
+
+public class TestBase {
+
+    Calculator calculator;
+
+    @BeforeMethod(alwaysRun = true)
+    public void setUp() {
+        calculator = new Calculator();
+    }
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw1/TestBase.java:14]: add test teardown

File: src/test/java/com/epam/tc/hw1/TestBase.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.BeforeMethod;
+
+public class TestBase {
+
+    Calculator calculator;
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw1/TestBase.java]: lets make it protected

**Review by ZeroShadows — CHANGES_REQUESTED** [review]

File: src/test/java/com/epam/tc/hw1/TestBase.java (hunk 1 above)
Comment (kalimullina) [reply src/test/java/com/epam/tc/hw1/TestBase.java:14]: fixed

File: src/test/java/com/epam/tc/hw1/TestBase.java (hunk 2 above)
Comment (kalimullina) [reply src/test/java/com/epam/tc/hw1/TestBase.java]: corrected

**Review by ZeroShadows — APPROVED** [review]

**Pull request merged**